packages/labs/src/components/tag-input/tagInput.tsx

```diff
@@ -48,6 +51,9 @@ export interface ITagInputProps extends IProps {
      */
     onAdd?: (values: string[]) => boolean | void;
 
+    /** Callback invoked when the value of `<input>` element is changed */
```

Comment:
trailing period, "This is shorthand for `inputProps={{ onChange }}`."

Reply:
Done